src/Select/Select.js

```diff
@@ -0,0 +1,188 @@
+  display: flex;
+  align-items: center;
+  position: absolute;
+  padding: ${({ theme }) => `0 ${theme.spaceSmall}`};
+  pointer-events: none;
+`;
+
+const SelectSizeCondition = (theme, size) => {
+  if (size === SIZE_OPTIONS.SMALL) {
+    return `${theme.spaceXSmall} ${theme.spaceXLarge} ${theme.spaceXSmall} ${theme.spaceSmall}`;
+  }
+  return `${theme.spaceSmall} ${theme.spaceXXLarge} ${theme.spaceSmall} ${theme.spaceMedium}`;
+};
+
+const StyledSelect = styled(({ theme, prefix, ...props }) => <select {...props} required />)`
+  width: 100%;
+  height: 100%;
+  border: none;
+  border-radius: ${({ theme }) => theme.borderRadiusNormal};
+  font-family: ${({ theme }) => theme.fontFamily};
+  font-size: ${({ theme, size }) =>
+    size === SIZE_OPTIONS.SMALL ? theme.fontSizeInputSmall : theme.fontSizeInputNormal};
+  color: ${({ theme }) => theme.colorTextInput};
+  background: ${({ theme }) => theme.backgroundInput};
+  padding: ${({ theme, size }) => SelectSizeCondition(theme, size)};
+  padding-left: ${({ prefix }) => prefix && "48px"}; // temporary replacement for token
+  appearance: none;
+  outline: none;
+  cursor: pointer;
+  /* for IE10 */
+  &::-ms-expand {
+    display: none;
+  }
+  &:disabled {
+    color: ${({ theme }) => theme.colorTextInputDisabled};
+    background: ${({ theme }) => theme.backgroundInputDisabled};
+    cursor: default;
+  }
+  /* placeholder */
+  &:invalid {
+    color: ${({ theme }) => theme.paletteInkLight};
+  }
+`;
+
+const SelectSuffix = styled(({ theme, disabled, ...props }) => <div {...props} />)`
+  position: absolute;
+  right: ${({ theme }) => theme.spaceXSmall};
+  color: ${({ theme, disabled }) =>
+    !disabled ? theme.colorTextInput : theme.colorTextInputDisabled};
+  pointer-events: none;
+  & > * {
+    width: ${({ theme, size }) => size === SIZE_OPTIONS.SMALL && theme.widthIconSmall};
+    height: ${({ theme, size }) => size === SIZE_OPTIONS.SMALL && theme.heightIconSmall};
+    margin-bottom: ${({ size }) =>
+      size === SIZE_OPTIONS.SMALL && "2px"}; // temporary replacement for token spaceXXXSmall
+  }
+`;
+
+class Select extends React.Component<Props, State> {
+  static defaultProps = {
+    value: "",
+    size: SIZE_OPTIONS.NORMAL,
+    disabled: false,
+    theme: defaultTokens,
+  };
+
+  state = {
+    value: this.props.value,
+  };
+
+  handleChange = (e: SyntheticInputEvent<HTMLSelectElement>) => {
+    this.setState({ value: e.target.value });
+    this.props.onChange(e);
+  };
+
+  renderOption = ({ value, label, visible = true }: Option) => {
+    if (visible) {
+      return (
+        <option key={`option-${value}`} value={value} label={label || value}>
+          {label || value}
+        </option>
+      );
+    }
+    return null;
+  };
+
+  render() {
+    const { theme, prefix, label, size, disabled, placeholder, help, error, options } = this.props;
+    const { value } = this.state;
+    return (
+      <React.Fragment>
+        {label && <Text>{label}</Text>}
```

Comment:
I'm completely missing `label` knob in stories. Don't forget that label's color is changing on the selected value.